### PR TITLE
Allows nil encrypted floats instead of defaulting to 0.0

### PR DESCRIPTION
**Pull request by mattevans**

### CODE REVIEW

**Comment by mattevans** [comment]
@snikch Any luck with reviewing/approving this PR mate?

File: lynx/float_test.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func TestScan(t *testing.T) {
 		}
 
 		f, err := ef.Float64()
-		if err != nil {
+		if err != nil && e.Type != "nil" {
```
Comment (snikch) [inline lynx/float_test.go:94]: Maybe add some comments as to why this is needed

**Comment by snikch** [comment]
👍

**Comment by mattevans** [comment]
Thanks bro! Will add this morning and then merge.

**Pull request merged**